brainstormforce/ultimate-addons-for-gutenberg
# Fix: Invalid block issue from core widget blocks

[Trello](https://trello.com/c/c5zfPzCo/275-fix-when-we-activate-uag-plugin-archive-block-generates-invalid-block-error)

### Description
Issue --> https://github.com/brainstormforce/ultimate-addons-for-gutenberg/issues/549 
Some lifterLMS --> https://github.com/gocodebox/lifterlms-blocks/issues/85

If the UAG plugin is activated and when we drag-drop core widget block into the page we get --> https://share.getcloudapp.com/nOuoDOWb

I have excluded the following blocks - 
'core/archives',
'core/calendar',
'core/latest-comments',
'core/tag-cloud',
'core/rss' 

### Test Cases
- Create a page
- Add widget blocks to the editor (from the WordPress core blocks lib)
- See an invalid block error in the console.

### Types of changes
- Excluded the blocks from extension attributes.

### How has this been tested?
Checked whether it is working properly or not on editor.

### Checklist:
